netcode.c

```diff
@@ -3754,6 +3758,61 @@ void netcode_server_send_client_packet( struct netcode_server_t * server, void *
     server->client_last_packet_send_time[client_index] = server->time;
 }
 
+/**
+ * @brief skillz_print_all_matches
+ *
+ * Mostly just a test function.  Prints every match in the hash.
+ *
+ */
+void skillz_print_all_matches( struct netcode_server_t * server )
+{
+    skillz_match_t * match;
+
+    netcode_printf( NETCODE_LOG_LEVEL_INFO, "\n\nPrinting the matches and their clients.\n" );
+    int i;
+    for( match = server->skillz_matches; match != NULL; match = ( skillz_match_t * )( match->hh.next ) )
+    {
+        for( i = 0; i < match->num_clients_in_match; ++i)
+        {
+            netcode_printf( NETCODE_LOG_LEVEL_INFO, "match id: %d client id: %d clients in match: %d\n",
+                    match->skillz_match_id, match->clients_in_match[i], match->num_clients_in_match );
+        }
+    }
+    netcode_printf( NETCODE_LOG_LEVEL_INFO, "\n\n" );
+}
+
+/**
+ * @brief skillz_match_disconnect
+ * @param server
+ * @param client_index
+ * @return bool if success.
+ *
+ * Removes the match from the hash table, frees the memeory.
+ */
+int skillz_match_disconnect( struct netcode_server_t * server, int client_index )
```

Comment:
nit but couldn't you return a `bool` (or equiv) instead of the 1/0?

Reply:
There are no bools in these parts.

Reply:
As in its not a built in type in C